src/components/MovieForm/MovieForm.stories.jsx

```diff
@@ -0,0 +1,27 @@
+import MovieForm from "./MovieForm";    
+import "./style.css";
+
+export default {
+    title: "MovieForm",
+    component: MovieForm,
+}
+
+const info = {
+    id: 1,
+    imageUrl: "./images/Pulp fiction.jpg",
+    name: "Pulp fiction",
+    releaseYear: 1995,
+    movieUrl: "https://movies.com/pulpfiction.html",
+    rating: 7.2,
+    genre: "Criminal",
+    duration: "2 hours 34 minutes",
+    description: "Jules Winnfield (Samuel L. Jackson) and Vincent Vega (John Travolta) are two hit men who are out to retrieve a suitcase stolen from their employer, mob boss Marsellus Wallace (Ving Rhames). Wallace has also asked Vincent to take his wife Mia (Uma Thurman) out a few days later when Wallace himself will be out of town. Butch Coolidge (Bruce Willis) is an aging boxer who is paid by Wallace to lose his fight. The lives of these seemingly unrelated people are woven together comprising of a series of funny, bizarre and uncalled-for incidents.—Soumitra",
+}
+
+const Template = (args) => <MovieForm {...args} />
+
+export const Default = Template.bind({});
+Default.args = {
+    movieInfo: info,
+    onSubmit: (event) => event.preventDefault(),
```

Comment:
I faced an error when I checked this storybook template (please look at the picture). I tried to click on Submit button.

![2023-04-30_20h34_00](https://user-images.githubusercontent.com/47754466/235367829-ccc5efe4-a6b1-4ba6-8542-d96ee3bd0248.png)

I guess you wait for the onSubmit function with for value argument (this is not an event object). So let's rewrite this like:
```
Default.args = {
    movieInfo: info,
    onSubmit: (formValue) => {
        console.log(formValue);
    },
}
```